connect/sdk/javascript-example/main.js

```diff
@@ -9,6 +9,7 @@ const texture = createConnectSession({
     referenceId: "connect-sdk-example",
     clientName: "Connect SDK Example",
     tags: ["connect-sdk-example"],
+    manufacturerFilters: { manufacturers: ['honeywell', 'daikin'] },
```

Comment:
Fyi, looks like the javascript-example changes are in here too. Think if ya merge the other PR though, the diff to main should remove these

Reply:
correct, I am going to merge them in order, just being lazy 👍